movies/2018/ocean-8.json

```diff
@@ -0,0 +1,27 @@
+{
+    "name": "Ocean's 8",
+    "year": 2018,
+    "director": "Gary Ross",
+    "writer": [
+      "Gary Ross",
+      "Olivia Milch"
+    ],
+    "actors": [
+        "Sandra Bullock",
+        "Cate Blanchett",
+        "Anne Hathaway",
+        "Mindy Kaling",
+        "Sarah Paulson",
+        "Awkwafina",
+        "Rihanna",
+        "Helena Bonham Carter"
+    ],
+    "plot": "Debbie Ocean gathers an all-female crew to attempt an impossible heist at New York City's yearly Met Gala.",

```

Comment:
should be storyline.

Reply:
<img width="828" alt="Screenshot 2019-10-10 at 15 27 14" src="https://user-images.githubusercontent.com/44982724/66578149-7a2bca00-eb72-11e9-9687-1f920bec65c8.png">
 Please can you make your json follow the same structure as ^

Reply:
As request, it was solved.